### ISSUE TITLE
Signpost trigger is stealing focus by default

### ISSUE THREAD

**Issue by Shijir** [vmware/clarity#3767]
## Describe the bug

The signpost trigger is stealing focus when it's initialized.

## How to reproduce

https://stackblitz.com/edit/signpost-trigger-stealing-focus

I think the culprit is in `clrIfOpen` because it looks like the directive has the same name input
```
@Input('clrIfOpen')
  public set open(value: boolean) {
    this.ifOpenService.open = value;
  }
```
So just by using this directive, we would set the value to the `open` property of `ifOpenService`.

## Expected behavior

The signpost trigger shouldn't set focus on itself when it's initialized.
It should get the focus only after its popover content gets closed.

## Versions

**App**

* Angular: [8.x.x]
* Node: [e.g. 8.10.0]
* Clarity: [2.x.x]

**Device:**

* Type: [e.g. MacBook]
* OS: [e.g. iOS]
* Browser [e.g. chrome, safari]
* Version [e.g. 22]

**Comment by Shijir** [comment]
Another issue I noticed is that when there are multiple signpost triggers, and I click on one of them after clicking on another, the focus is going back to the inactive signpost trigger.

![Screen Shot 2019-08-30 at 7 34 51 AM](https://user-images.githubusercontent.com/3958008/64029078-b8a48280-caf8-11e9-95cf-1c8a2a79efa7.png)

**Comment by Shijir** [comment]
Closed by #3774
